src/Service/EventRejectedEmailService.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Joindin\Api\Service;
+
+use DateTime;
+
+class EventRejectedEmailService extends BaseEmailService
```

Comment:
This class doesn't have any tests, can we add them?

Reply:
[![codecov](https://codecov.io/gh/joindin/joindin-api/branch/master/graph/badge.svg)](https://codecov.io/gh/joindin/joindin-api)